chamber/data_transfer.py

```diff
@@ -1,11 +1,39 @@
+import pandas as pd
 from os import listdir
 from re import compile
+from nptdms import TdmsFile
+
+FILE_PATH = ""
 
 def list_tdms(file_path):
     """Returns a list of the .tdms files contained within the argument file."""
+    FILE_PATH = file_path
     file_list = []
     regex = compile(".tdms")
     for file in listdir(file_path):
         if regex.match(file, len(file) - len(".tdms")):
             file_list.append(file)
     return file_list
+
+def get_settings(tdms_obj):
```

Comment:
It may be better to pass the pandas dataframe into this function rather than creating it when it runs. This is because that dataframe will be used again and would probably be created outside of this function. We can talk about this more, but perhaps we don't even need a pandas dataframe. We may be able to just use the tdms object. Let's chat about this in the lab.